### ISSUE TITLE
Allow deploying from tags

### ISSUE THREAD

**Issue by LittleFox94** [actions/deploy-pages#36]
Hi,

I'm trying to deploy pages via this action from a workflow started for a pushes, both branches and tags. I configured the branch and tags as protected and the `github-pages` environment for `Deployment branches = Protected branches`. First problem: there is no selection of `Protected branches and tags`.

Interestingly, if I add a **branch** protection rule matching the tags (`v*.*.*`), this works - applying the branch protection rule to the tag and thus allowing the deployment via tag-push-triggered workflow.

**TL;DR**: allow `github-pages` environment to be updated from (protected) tags, too, and do not apply branch protection rules to tags.

**Comment by yoannchaudet** [comment]
Hello, a good place for this request would be the [community forum](https://github.community).
